src/client/components/UsersListComponent/UsersList.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './UsersListStyle.css';
+
+export default function Userslist(props) {
+  return (
+    <div className="container">
+      <div className="wraper">
+        <ul>
+          {props.people.map((people) => (
+            <li key={people}>{people}</li>
+          ))}
+        </ul>
+      </div>
+    </div>
+  );
+}
+
+Userslist.propTypes = {
+  people: PropTypes.arrayOf(PropTypes.string).isRequired,
```

Comment:
I would rename `people` to users`

```suggestion
  users: PropTypes.arrayOf(PropTypes.string).isRequired,
```